src/JoomlaBrowser.php

```diff
@@ -204,18 +204,9 @@ public function installJoomlaRemovingInstallationFolder()
 
         $this->debug('Removing Installation Folder');
         $I->click(['xpath' => "//input[@value='Remove installation folder']"]);
+
         $I->debug('I wait for Removing Installation Folder button to become disabled');
-        // @todo https://github.com/joomla-projects/joomla-browser/issues/45
-        $I->wait(2);
-        /*
-        $I->waitForElementChange(
-            ['xpath' => "//input[@name='instDefault']"],
-            function(WebDriverElement $el) {
-                return !$el->isEnabled();
-            },
-            60
-        );
-        */
+        $I->waitForJS("return jQuery('form#adminForm input[name=instDefault]').attr('disabled') == 'disabled';", 60);
```

Comment:
@compojoom if you are trying to execute the click event of the "remove installation folder" button through javascript, then do you think we still need https://github.com/joomla-projects/joomla-browser/pull/74/files#diff-676cca4b1f4fdd04825e9c7ae7b54691R206